tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -80,18 +80,18 @@ def montecarlo_radial1d(model, plasma, runner):
         runner.virt_packet_energies = np.concatenate(
             np.array(virt_packet_energies)
         ).ravel()
-        runner.virt_packet_last_interaction_in_nu = np.array(
-            virt_packet_last_interaction_in_nu
-        )
-        runner.virt_packet_last_interaction_type = np.array(
-            virt_packet_last_interaction_type
-        )
-        runner.virt_packet_last_line_interaction_in_id = np.array(
-            virt_packet_last_line_interaction_in_id
-        )
-        runner.virt_packet_last_line_interaction_out_id = np.array(
-            virt_packet_last_line_interaction_out_id
-        )
+        runner.virt_packet_last_interaction_in_nu = np.concatenate(
+            np.array(virt_packet_last_interaction_in_nu)
+        ).ravel()
+        runner.virt_packet_last_interaction_type = np.concatenate(
+        np.array(virt_packet_last_interaction_type)
+        ).ravel()
+        runner.virt_packet_last_line_interaction_in_id = np.concatenate(
+        np.array(virt_packet_last_line_interaction_in_id)
+        ).ravel()
+        runner.virt_packet_last_line_interaction_out_id = np.concatenate(
+        np.array(virt_packet_last_line_interaction_out_id)
```

Comment:
Seems like you don't have black enabled - we can ignore it for now though! :joy: